extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/ItagItem.java

```diff
@@ -57,6 +57,8 @@ public class ItagItem implements Serializable {
             new ItagItem(249, AUDIO, WEBMA_OPUS, 50),
             new ItagItem(250, AUDIO, WEBMA_OPUS, 70),
             new ItagItem(251, AUDIO, WEBMA_OPUS, 160),
+            new ItagItem(599, AUDIO, M4A, 32),
+            new ItagItem(600, AUDIO, WEBMA_OPUS, 48),
```

Comment:
Itag 600 is not a 48k audio, that's the itag 249. Instead, it is a 30-35-40k audio.

Reply:
@AudricV Thanks for your review! I have updated the PR.
Please let me know if I should submit another PR to the app to whitelist the new formats now or after merging this one.